R/config-defaults.R

```diff
@@ -230,6 +230,33 @@ config <- list(
     )
   },
 
+  ppm.enabled = function(..., default = TRUE) {
+    renv_config_get(
+      name    = "ppm.enabled",
+      type    = "logical[1]",
+      default = default,
+      args    = list(...)
+    )
+  },
+
+  ppm.default = function(..., default = TRUE) {
```

Comment:
Maybe we should the default as `FALSE` for macs?

Reply:
The package manager team does intend to make binaries available for at least some recent versions of R on macOS, so I'm hoping we'll be able to just turn this on by default for all platforms.

Reply:
I'm just worried that in the short term it's going to make testing renv more of a pain for you + me.